gatsby-config.ts

```diff
@@ -104,6 +104,15 @@ const config: GatsbyConfig = {
         },
       },
     },
+    // robots.txt creation
+    {
+      resolve: "gatsby-plugin-robots-txt",
+      options: {
+        host: "https://ethereum.org/",
+        sitemap: "https://ethereum.org/sitemap/sitemap-index.xml",
```

Comment:
There is a local variable called `siteUrl` at the beginning of this file. Lets try to use it here just as a good practice. What do you think?

Reply:
Of course I'm ok, I missed that !